notebooks/224-3D-segmentation-point-clouds/224-3D-segmentation-point-clouds.ipynb

```diff
@@ -0,0 +1,247 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part Segmentation of 3D Point Clouds with OpenVINO™  \n",
+    "\n",
+    "This notebook demonstrates how to process [point cloud](https://en.wikipedia.org/wiki/Point_cloud) data and run 3D Part Segmentation with OpenVINO. We use the [PointNet](https://arxiv.org/abs/1612.00593) pre-trained model to detect each part of a chair and return its category.\n",
+    "## PointNet\n",
+    "PointNet was proposed by Charles Ruizhongtai Qi, a researcher at Stanford University in 2016: arXiv:1612.00593 <[PointNet: Deep Learning on Point Sets for 3D Classification and Segmentation](https://arxiv.org/abs/1612.00593)>. The motivation behind the research is to classify and segment 3D representations of images. They use a data structure called point cloud, which is a set of points that represents a 3D shape or object. PointNet provides a unified architecture for applications ranging from object classification, part segmentation, to scene semantic parsing. It is highly efficient and effective, showing strong performance on par or even better than state of the art. \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "from pathlib import Path\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from openvino.runtime import Core\n",
+    "\n",
+    "sys.path.append(\"../utils\")\n",
+    "from notebook_utils import download_file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare the Model\n",
+    "- Download the pre-trained PointNet ONNX model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the data and model directories, model source URL and model filename\n",
+    "MODEL_DIR = \"model\"\n",
+    "os.makedirs(MODEL_DIR, exist_ok=True)\n",
+    "download_file(\"https://storage.googleapis.com/ailia-models/pointnet_pytorch/chair_100.onnx\", directory=Path(MODEL_DIR), show_progress=False)\n",
+    "onnx_model_path = Path(MODEL_DIR) / \"chair_100.onnx\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- Convert the ONNX model to OpenVINO IR"
```

Comment:
"IR" is the 1st time appeared in this notebook. Could you also provide its full name, and also a bit explanation of what the ".xml" and ".bin" standard for? In case the develper is not very familiar with OV format.

Reply:
thanks for your review